From 7b2495a4634987793b592f219a0c80c394c40d70 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 25 Jan 2023 14:54:28 +0100
Subject: [PATCH] arm64: defconfig: enable rpmsg-irq in addons fragment

Add the support of the IRQ controller over rpmsg.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Ia1ab4d04bac4d512e1c32d39ef42e505cf754b95
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310429
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 75e1d0ae0e6b..9155be0bf2cf 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -23,6 +23,7 @@ CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REMOTEPROC=y
 CONFIG_RPMSG_CHAR=m
 CONFIG_RPMSG_CTRL=m
+CONFIG_RPMSG_IRQ=m
 CONFIG_RPMSG_TTY=m
 CONFIG_RPMSG_VIRTIO=m
 CONFIG_RTC_DRV_STM32=y
-- 
2.39.5

